source/quickstart.txt

```diff
@@ -119,7 +119,7 @@ Setup
 
    .. step:: Download and install ``mongosync``
 
-      :ref:`c2c-mongosync` is the tool that connects the source and
+      :ref:`mongosync <about-mongosync>` is the tool that connects the source and
```

Comment:
In general, let's [ minimize repeated links](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#minimize-repeated-links).  Can you go through the pages you've edited and make sure they follow this style guideline to avoid linking to the same target multiple times in the same section. Since this tutorial already links to the mongosync behavior page, I think we can monospace this mention of mongosync (and mentions later down the page)

Reply:
Fixing now and also changing all ":ref:`mongosync <about-mongosync>`" to ":program:`mongosync`" (links to command line option page)